circe.el

```diff
@@ -1489,6 +1500,20 @@ PATTERNS should be the list of regular expressions."
           (throw 'return t)))
       nil)))
 
+(defun circe-tracking-get-face (buffer)
+  "Return face for a given buffer. If the buffer has already some face
+specified, use it. This covers channels where our name was mentioned. Otherwise
+decide whether the buffer is for a channel, query or else and use a face based
+on this."
+  (with-current-buffer buffer
+    (cond ((get-text-property 0 'face buffer)
+           (get-text-property 0 'face buffer))
+          ((eq major-mode 'circe-channel-mode)
+           'circe-tracking-channel-face)
+          ((eq major-mode 'circe-query-mode)
+           'circe-tracking-query-face)
+          (tracking-get-facet nil))))
```

Comment:
Typo. It's unclear how this is supposed to work as there's no such variable.

Reply:
Ah, sorry. It was supposed to be `tracking-get-face` from `tracking.el`. Updated.